src/main.cpp

```diff
@@ -568,11 +567,6 @@ namespace {
 		return "";
 	}
 
-#ifdef _WIN32
-	const char *debuggerNames[] = {"gdb.exe", "lldb.exe"};
-#else
-	const char *debuggerNames[] = {"gdb", "lldb"};
-#endif
```

Comment:
I'd prefer `debuggerNames` to stay where it is, since `getDebuggerArgs` below depends on the order of the array.

Reply:
Alright.